src/main/java/com/google/firebase/database/tubesock/WebSocket.java

```diff
@@ -44,6 +46,9 @@
  */
 public class WebSocket {
 
+  private static final ThreadConfig DEFAULT_THREAD_CONFIG = new ThreadConfig(
+      ThreadInitializer.defaultInstance, Executors.defaultThreadFactory());
```

Comment:
Should this get the thread initializer from Platform.java?

Reply:
This constant is not used during normal execution. It is only used in the hidden `Websocket(URL url)` constructor, which is only called by some internal test classes we have implemented (a set of executable `main` methods!). So I believe this is ok. 

We can discuss later whether we need this at all. I think this and the test client code that use it can all be removed.